infrastructure/helm-chart/charts/apps/charts/media-resolver/values.yaml

```diff
@@ -1,8 +1 @@
 image: media/resolver
-storage:
```

Comment:
This should prevent the `media-resolver` from starting by default. Not the full fix for #1497, but improves the situation.